lambda/custom/index.js

```diff
@@ -2,250 +2,7 @@
-      console.log(err.message);
-      return `Sorry, I couldn't send your message right now`;
-    });
-};
-
-const channelLastMessage = async (channelName, headers) => {
-  return await axios.get(`${channelmessageurl}${channelName}`,
-    { headers: headers })
-    .then(res => res.data)
-    .then(res => {
-      if(res.success == true){
-        return `${res.messages[0].u.name} says, ${res.messages[0].msg}`;
-      }
-      else{
-        return `Sorry, I couldn't find the channel ${channelName} right now`;
-      }
-    })
-    .catch(err => {
-      console.log(err.message);
-      if(err.response.data.errorType == `error-room-not-found`){
-        return `Sorry, the channel ${channelName} does not exist. Please try again with different channel name.`;
-      }
-      else{
-        return `Sorry, I couldn't find the channel ${channelName} right now`;
-      }
-    });
-};
-
-const getUserInfo = async (username0) => {
-  try {
-    const { data } = await axios({
-      method: 'get',
-      url: userinfourl + username0,
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot get channel message', error);
-    return error;
-  }
-};
-
-const getRoomInfo = async (channelname0) => {
-  try {
-    const { data } = await axios({
-      method: 'get',
-      url: channelinfourl + channelname0,
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot get channel message', error);
-    return error;
-  }
-};
-
-
-const addAll = async (roomid) => {
-  try {
-    const { data } = await axios({
-      method: 'post',
-      url: addallurl,
-      data: {
-        roomId: roomid
-      },
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot add all', error);
-    return error;
-  }
-};
-
-const makeModerator = async (userid, roomid) => {
-  try {
-    const { data } = await axios({
-      method: 'post',
-      url: makemoderatorurl,
-      data: {
-        userId: userid,
-        roomId: roomid
-      },
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot make moderator', error);
-    return error;
-  }
-};
-
-const addOwner = async (userid, roomid) => {
-  try {
-    const { data } = await axios({
-      method: 'post',
-      url: addownerurl,
-      data: {
-        userId: userid,
-        roomId: roomid
-      },
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot make owner', error);
-    return error;
-  }
-};
-
-const archiveChannel = async (roomid) => {
-  try {
-    const { data } = await axios({
-      method: 'post',
-      url: archivechannelurl,
-      data: {
-        roomId: roomid
-      },
-      headers: headers
-    });
-    return data;
-  } catch (error) {
-    console.error('cannot archive channel', error);
-    return error;
-  }
-};
-
+const functions_axios = require('./functions_axios');
```

Comment:
Also, prefer naming it something like "helper functions"
because we might use more functions there which are not related to axios.